### PR TITLE
Updating the scripts everyday midnight

### PR DESCRIPTION
**Pull request by Basazababo**

### CODE REVIEW

File: emoncmsupdate [hunk 1]
```diff
@@ -2,38 +2,38 @@
...
+echo "Emonhub update started"
 echo 
 
 date
 
 echo
 
-cd /var/www/emoncms
-git pull origin low-write
+#cd /var/www/emoncms
```
Comment (jlark) [inline emoncmsupdate:12]: It look slike  emoncms is still updating??

File: emoncmsupdate [hunk 2]
```diff
@@ -2,38 +2,38 @@
...
+#git pull origin master
 
-cd /var/www/emoncms/Modules/wifi
-git pull origin master
+#cd /var/www/emoncms/Modules/wifi
+#git pull origin master
 
-cd /home/pi/emonhub
+cd /home/debian/emonhub
 git pull origin emon-pi
```
Comment (jlark) [inline emoncmsupdate]: should we be pulling beaglebone branch?

**Comment by Basazababo** [comment]
I think we need to pull GLE/emonhub repo for emonhub update, am i correct?
i tested with git pull origin emon-pi and it works

**Comment by jlark** [comment]
emon-pi is the branch though. YOu should be using the beaglebone branch!

**Comment by Basazababo** [comment]
Cool, That`s true

**Comment by Basazababo** [comment]
"It look like emoncms is still updating??" you said.
I disabled that, do you think there is other place RMC is updating this?

**Comment by jlark** [comment]
ok nevermind, I just saw that you commented it out.

**Comment by jlark** [comment]
No I didn't see your commented them out. That what I was referring to.

**Comment by jlark** [comment]
ok look good to me.
Is it tested?

**Comment by Basazababo** [comment]
Yeap,
I tested it,

**Pull request merged**